extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItem.java

```diff
@@ -18,9 +18,7 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-/**
- Media item returned by queries to the Google Photos API.  Represents what is stored by Google.
- */
+/** Media item returned by queries to the Google Photos API. Represents what is stored by Google. */
```

Comment:
Can you please reverse these formatting changes on the pull request. Unfortunately it makes the pull request harder to review and also pollutes the git history which makes tools like blame less helpful.

Reply:
Agreed, this makes sense. We may want to add a step to the Travis CI to check this. I have created https://github.com/google/data-transfer-project/issues/739 to track this